### PR TITLE
Pg migrate

### PR DESCRIPTION
**Pull request by sohilpandya**
- updated pg-migrate script
- added pg-native module for pg-migration on production with ssl: true

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "src/server/server",
   "scripts": {
     "start": "node src/server/start.js",
-    "db-migrate-up": "node_modules/.bin/db-migrate up --config database.json",
+    "db-migrate-up": "node_modules/.bin/db-migrate up --config database.json -e prod",
```
Comment (emilyb7) [inline package.json:8]: what if we're not doing the migration on production? would this still work?

**Review by emilyb7 — COMMENTED** [review]
Hey! So I'm not 100% clear about what the workflow would be for doing DB migrations either locally or in production. Do we have to run the script manually or does it get done automatically when we deploy? Can we clarify this in the readme?

**Comment by sohilpandya** [comment]
@emilyb7 updated readme with some details.

**Comment by emilyb7** [comment]
nice, that was fast! 🚀🤘🏻✨

**Review by emilyb7 — APPROVED** [review]

**Pull request merged**